androidApp/src/main/java/com/ajay/seenu/expensetracker/android/presentation/common/MultiSelectChipView.kt

```diff
@@ -0,0 +1,150 @@
+        ) {
+            onClick.invoke()
+        }
+        .border(1.dp, borderColor, RoundedCornerShape(10.dp))) {
+        Row(modifier = Modifier.fillMaxWidth()
+            .padding(top = 10.dp)) {
+            Text(
+                text = "Attachments (${selectedOptions.size})",
+                modifier = Modifier.weight(1f).padding(start = 10.dp)
+            )
+            Icon(
+                modifier = Modifier.padding(end = 15.dp),
+                painter = painterResource(id = expandIcon), contentDescription = "drop down"
+            )
+        }
+        Row(modifier = modifier
+            .fillMaxWidth()
+            .heightIn(MinContainerHeight, MaxFlowRowHeight)
+            .verticalScroll(scrollState),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Start) {
+            FlowRow(
+                horizontalArrangement = Arrangement.spacedBy(7.dp),
+                verticalArrangement = Arrangement.spacedBy(7.dp),
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(10.dp)
+            ) {
+                selectedOptions.forEach {
+                    InputChip(
+                        modifier = Modifier,
+                        selected = false,
+                        enabled = true,
+                        onClick = {},
+                        label = {
+                            selectionOptionView.invoke(it)
+                        },
+                        trailingIcon = {
+                            Box(
+                                modifier = Modifier
+                                    .size(20.dp)
+                                    .clip(RoundedCornerShape(5.dp))
+                                    .background(chipTrailingIconBackgroundColor)
+                                    .clickable {
+                                        onOptionCanceled.invoke(it)
+                                    },
+                                contentAlignment = Alignment.Center
+                            ) {
+                                Icon(
+                                    painter = painterResource(id = chipTrailingIcon),
+                                    contentDescription = "cancel",
+                                    tint = chipTrailingIconColor
+                                )
+                            }
+                        },
+                        colors = InputChipDefaults.inputChipColors(
+                            containerColor = chipContainerColor
+                        ),
+                        border = InputChipDefaults.inputChipBorder(
+                            enabled = true,
+                            selected = false,
+                            borderColor = chipBorderColor
+                        )
+                    )
+                }
+            }
+        }
+    }
+}
+
+private val MinContainerHeight = 40.dp
```

Comment:
It is very specific to that view da. I guess that it will not be reusable. If we get a case for custom container height in future, we can move it to Defaults along with colors and icons. Let it be private for now